packages/application-components/src/components/custom-views/custom-views-selector/use-custom-views-connector.ts

```diff
@@ -21,6 +23,10 @@ type TUseCustomViewsFetcher = (props: TUseCustomViewsFetcherParams) => {
 export const useCustomViewsConnector: TUseCustomViewsFetcher = ({
   customViewLocatorCode,
 }) => {
+  const areCustomViewsEnabled =
+    useFeatureToggle(featureFlags.CUSTOM_VIEWS) &&
+    process.env.DISABLE_CUSTOM_VIEWS_FEATURE !== 'true';
```

Comment:
This is to easily be able to disable the feature when testing consumer applications so the request to fetch the Custom Views will not be sent.